components/Form/Form.jsx

```diff
@@ -0,0 +1,147 @@
+'use client'
+import { Button } from '../ui/Button/Button'
+import React, { useState } from 'react'
+
+const returnFormElement = function (columnConfig) {
+  switch (columnConfig.cellType) {
+    case 'text':
+      return (
+        <div
+          key={columnConfig.accessorKey}
+          className="w-240 md:w-1/2 px-3 mb-6 md:mb-0">
+          <label
+            className="block uppercase tracking-wide text-gray-700 text-xs font-bold mb-2"
+            htmlFor="grid-first-name">
+            {columnConfig.header}
+          </label>
+          <input
+            key={columnConfig.id}
+            className="appearance-none block w-full bg-gray-200 text-gray-700 border border-gray-200 rounded py-3 px-4 mb-3 leading-tight focus:outline-none focus:bg-white"
+            id="grid-first-name"
+            type="text"
+            placeholder="Jane"
+          />
+        </div>
+      )
+      break
```

Comment:
You can check the input component into ui and do the same with select and multi-select component. You can use the [radix-ui component](https://www.radix-ui.com/themes/docs/components/select). The main advantage to base your component on it, radix manage accessibility out of the box for you